hosted/gke/p0/p0_suite_test.go

```diff
@@ -68,18 +67,20 @@ var _ = ReportAfterEach(func(report SpecReport) {
 	Qase(testCaseID, report)
 })
 
-func p0upgradeK8sVersionCheck(cluster *management.Cluster, client *rancher.Client, clusterName string) {
```

Comment:
Explicitly passing the `client *rancher.Client` was intentional, I would prefer to not use global variables such as `ctx` inside these check functions.